addons/spreadsheet/static/src/global_filters/helpers.js

```diff
@@ -22,18 +18,25 @@ export function checkFiltersTypeValueCombination(type, value) {
                     return CommandResult.InvalidValueTypeCombination;
                 }
                 break;
-            case "date":
-                if (typeof value === "string") {
+            case "date": {
+                if (value === "") {
+                    return CommandResult.Success;
```

Comment:
this is a bit stupid, but I keep the empty string as a valid value for backward compatibility.